katas/7kyu/reverse-words.js

```diff
@@ -0,0 +1,10 @@
+function solution(str, ending) {
+    if (ending === '') return true;
```

Comment:
could be simplified to
```js
if (!ending) return true
```

Reply:
![4BoK](https://github.com/user-attachments/assets/05e9edaf-4601-4105-8f37-f1993b1eadf8)